modules/age.nix

```diff
@@ -16,13 +16,15 @@ let
 
   identities = builtins.concatStringsSep " " (map (path: "-i ${path}") cfg.sshKeyPaths);
   installSecret = secretType: ''
-    echo "decrypting ${secretType.file} to ${secretType.path}..."
-    TMP_FILE="${secretType.path}.tmp"
-    mkdir -p $(dirname ${secretType.path})
+    echo "decrypting '${secretType.file}' to '${cfg.secretsMountPoint}/$_count/${secretType.name}'..."
```

Comment:
Maybe create a var for `${cfg.secretsMountPoint}/$_count/${secretType.name}"` to make this (and the lines below) a bit more readable?

Reply:
WDYT of `local truePath="....."` or `local secretPath="....."`?